R_Friedmantest.qmd

```diff
@@ -0,0 +1,52 @@
+---
+title: "Friedman Chi-Square test using R package rstatix"
+---
+
+# **Example: Simulated dataset with 10 subjects each having results at each of the 4 time points.**
+
+Simulated datasets with continuous endpoints are generated for single-drug repeated measurements to check whether any significance exists between the responses at different time points. The p-value will indicate whether differences in response for different time points are significant.
+
+*Data source:*
+
+```{r, output = FALSE, message = FALSE}
+library(dplyr)
+library(rstatix) 
+```
+
+```{r eval=FALSE}
+set.seed(123)
+
+Onewayrepeat = data.frame(subject = rep(1:10, each=4),
+                          timepoint = rep(c(1, 2, 3, 4), times=10),
+                          Response =round(runif(n = 40, 10, 50)))
+
+head(Onewayrepeat)
+```
+
+## Overview
+
+The friedman.test() function from the R package 'rstatix' computes the following:
+
+-   Friedman chi-squared statistic
+
+-   its degrees of freedom and
+
+-   p-value for checking whether the mean response is the same for all the time points.
+
+**Missing Values**
+
+when the data contain NAs. the function Defaults to getOption("na.action") which discards the row with NAs
+
+## Example Code
+
+## **Freidman Chi-Square test**
+
+```{r eval=FALSE}
+friedman.test(y=Onewayrepeat$Response, groups=Onewayrepeat$timepoint, blocks=Onewayrepeat$subject)
+```
+
+where: y - a vector of response values. groups - a vector of values indicating the "group" an observation belongs in. blocks - a vector of values indicating the "blocking" variable.
```

Comment:
Perhaps earlier when you describe the simulated data generation you could relate it back to these params?